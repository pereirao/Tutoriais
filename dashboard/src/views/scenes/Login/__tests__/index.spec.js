import React from 'react';

import { Login } from '../index';

it('renders correctly', () => {
  expect(shallow(<Login is_logged_in={false} />)).toMatchSnapshot();
});
