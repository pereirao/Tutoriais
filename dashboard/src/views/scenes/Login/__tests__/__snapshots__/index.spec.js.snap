// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="wrapper"
  >
    <div
      className="slidingBackground"
    />
  </div>
  <div
    className="panelWrapper"
  >
    <div
      className="panel"
    >
      <MBForm
        className="formWrapper"
        field_spacer_width={16}
        submit_from_last={true}
        validateSubmittable={[Function]}
      >
        <img
          alt="Minibar Delivery Logo"
          className="logo"
          src="minibar_logo.png"
        />
        <EmailField
          is_required={true}
          name="email"
        />
        <PasswordField
          is_required={true}
          name="password"
          submit_on_enter={true}
        />
        <LoginError />
        <MBFormSubmitButton
          inverted={true}
          is_loading={false}
          onClick={[Function]}
          text="LOG IN"
          title="Submit form"
        />
        <a
          className="forgotPasswordLink"
          href="https://api.minibardelivery.com/users/password/new"
          rel="noopener noreferrer"
          target="_blank"
        >
          forgot password?
        </a>
      </MBForm>
    </div>
  </div>
</div>
`;
